.github/workflows/linting.yml

```diff
@@ -10,6 +10,8 @@ jobs:
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
+      - name: Ignore ui folder by removing it
+        run: rm -rf ./ui        
```

Comment:
Maybe also rm the /framework directory? Or do you want to wait a bit, since /framework will go away anyway.